src/build/__main__.py

```diff
@@ -42,6 +42,19 @@ def _error(msg, code=1):  # type: (str, int) -> None  # pragma: no cover
     exit(code)
 
 
+def _print_info(msg):  # type: (str) -> None
+    """
+    Custom build.print_info implementation to print directly to stdout, and with
+    colors.
+    """
+    prefix = '*'
+    end = ''
+    if sys.stdout.isatty():
+        prefix = '\33[1m\33[36m' + prefix + '\33[39m'
```

Comment:
If you're introducing colors we should respect ``NO_COLOR`` env var, and add a flag to disable it. It can be on by default.

Reply:
This color is only available in the main and is only active in a TTY. But I can add an env var if that really matters. We currently already have colors in main and nobody has asked for that. I believe checking for a TTY should handle 99.9% of the cases, a lot of other software does the same.

Reply:
Yeah, let's please follow https://no-color.org as does pytest, tox, etc.